github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPROpenEvent/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger when PR is opened or commits (hash) changed in it.
+    Triggers a build of a pull request when opened or its commits have changed.
```

Comment:
Btw, i splitted this event into 2 logical. This Event shouldn't rely on `hash changed`. Could you check java class? 
Use case: prepare special env only when PR was opened, without caring about hashes. PRCloseEvent then may be used for deleting special env.  Hash changes doesn't play in this party :)